skglm/estimators.py

```diff
@@ -126,7 +131,8 @@ def _glm_fit(X, y, model, datafit, penalty, solver):
             w = np.zeros(n_features + fit_intercept, dtype=X_.dtype)
             Xw = np.zeros(n_samples, dtype=X_.dtype)
         else:  # multitask
-            w = np.zeros((n_features + fit_intercept, y.shape[1]), dtype=X_.dtype)
+            w = np.zeros((n_features + fit_intercept,
```

Comment:
change introduced by autoformatting, we don't want to have them since they make the PR harder to review and increase the risk of conflict. 

Add this to your setting (either user, either workspace if you want them to hold only for this project: 
```
    "autopep8.args": [
        "--max-line-length=88"
    ],
```